package.json

```diff
@@ -9,20 +9,22 @@
     "version": "independent"
   },
   "scripts": {
+    "dev": "vite",
+    "serve": "vite preview",
```

Comment:
remove since we have the preview targets?
```suggestion
```

Reply:
As you want, could be useful to see another env in the end but happy to bring it back later if it's needed, removed in [780fc35](https://github.com/shapeshift/web/pull/8924/commits/780fc358dd698fa1d4f35579acc5f620dc5da98a)